nowcasting_dataloader/datasets.py

```diff
@@ -68,6 +69,7 @@ def __init__(
             cloud: which cloud is used, can be "gcp", "aws" or "local".
             normalize: normalize the batch data
             add_position_encoding: Whether to add position encoding or not
+            data_sources_names: Names of data sources to load, if not using all of them
```

Comment:
I might suggest making the docstring a tiny bit more explicit (and below, too, in `SatFlowDataset.__init__`), something like...

```suggestion
            data_sources_names: Names of data sources to load.  Leave as None to load all data sources.
```